app/src/test/java/org/simple/clinic/registration/phone/loggedout/LoggedOutOfDeviceDialogControllerTest.kt

```diff
@@ -32,23 +33,21 @@ class LoggedOutOfDeviceDialogControllerTest {
   private val userSession = mock<UserSession>()
   private val uiEvents = PublishSubject.create<UiEvent>()
 
-  private val controller = LoggedOutOfDeviceDialogController(userSession)
+  private lateinit var controllerSubscription: Disposable
 
-  @Before
-  fun setUp() {
-    RxJavaPlugins.setErrorHandler(null)
-
-    whenever(userSession.logout())
-        .thenReturn(Single.never())
-
-    uiEvents
-        .compose(controller)
-        .subscribe({ uiChange -> uiChange(dialog) }, { throw it })
+  @After
+  fun tearDown() {
+    controllerSubscription.dispose()
   }
 
   @Test
   fun `when the dialog is created, the okay button must be disabled`() {
+    // given
+    RxJavaPlugins.setErrorHandler(null)
```

Comment:
We can pass the error handler as a parameter to `setupController` and avoid setting this up in each test, right?

Reply:
Ah right, will do that in next PR.